net-im/signal-desktop-bin/signal-desktop-bin-1.37.2.ebuild

```diff
@@ -90,6 +93,11 @@ src_install() {
 	dodoc changelog
 	doins -r opt
 	insinto /usr/share
+
+	if has_version media-sound/apulse[-sdk]; then
+		sed -i 's/Exec=/Exec=apulse /g' usr/share/applications/signal-desktop.desktop
+	fi
```

Comment:
what if I happen to have both installed?

Reply:
You can't, they block each other.

Reply:
I adjusted this: if pulseaudio is installed we don't need to patch the desktop file.